packages/stokhos/src/sacado/kokkos/vector/Kokkos_View_MP_Vector_Interlaced.hpp

```diff
@@ -350,7 +350,7 @@ class View< DataType , Arg1Type , Arg2Type , Arg3Type , Impl::ViewMPVectorInterl
   //------------------------------------
   // Allocation of a managed view with possible alignment padding.
 
-  typedef Impl::if_c< traits::is_managed ,
+  typedef std::conditional< traits::is_managed ,
```

Comment:
`if_allocation_constructor` is not actually used anywhere.  If it was up to me I would just remove the definition.

Reply:
Fine with me.

Reply:
Want me to remove in this PR?

Reply:
Done